carbonserver/carbonserver/api/infra/repositories/repository_projects.py

```diff
@@ -43,20 +41,21 @@ def get_one_project(self, project_id) -> Project:
                 )
             return self.map_sql_to_schema(e)
 
-    def get_projects_from_team(self, team_id) -> List[Project]:
-        """Find the list of projects from a team in database and return it
+    # TODO: DO a version of this method but that returns a list of projects of an organization
+    # def get_projects_from_team(self, team_id) -> List[Project]:
+    #     """Find the list of projects from a team in database and return it
 
-        :team_id: The id of the team to retreive projects from.
-        :returns: List of Projects in pyDantic BaseModel format.
-        :rtype: List[schemas.Project]
-        """
-        with self.session_factory() as session:
-            res = session.query(SqlModelProject).filter(
-                SqlModelProject.team_id == team_id
-            )
-            if res.first() is None:
-                return []
-            return [self.map_sql_to_schema(e) for e in res]
+    #     :team_id: The id of the team to retreive projects from.
+    #     :returns: List of Projects in pyDantic BaseModel format.
+    #     :rtype: List[schemas.Project]
+    #     """
+    #     with self.session_factory() as session:
+    #         res = session.query(SqlModelProject).filter(
+    #             SqlModelProject.team_id == team_id
```

Comment:
This is the interesting filter. But remove this code instead of commenting it, we will know how to re-write it when needed

Reply:
done!